tests/e2e/no_hci_mesh_rwx_singlevc_topology.go

```diff
@@ -2227,8 +2227,8 @@ var _ = ginkgo.Describe("[rwx-nohci-singlevc-positive] RWX-Topology-NoHciMesh-Si
 		11. Perform cleanup by deleting Pod, PVC and SC.
 	*/
 
-	ginkgo.It("Create pvc with rox access mode and config secret "+
-		"netpermissions is set with readonly permissions", ginkgo.Label(p0, file, vanilla, level5, level2, newTest), func() {
+	ginkgo.It("Create pvc with rox access mode and config secret netpermissions is set with readonly "+
+		"permissions", ginkgo.Label(p0, file, vanilla, level5, level2, newTest, vc70), func() {
```

Comment:
Please add tkg label as well. Also, make sure ginkgo picks up right labels as we are adding label in both testcase level and file level.

Reply:
I meant in operation.go file

Reply:
tkg label not applicable here